employees/forms.py

```diff
@@ -22,3 +29,30 @@ class Meta:
         model = Report
         fields = ("date", "description", "task_activities", "project", "work_hours")
         widgets = {"date": DatePickerInput(format="%Y-%m-%d")}
+
+
+class MonthSwitchForm(forms.Form):
+
+    date = forms.DateField(
+        widget=DatePickerInput(format="%m-%Y"),
+        label=False,
+        validators=[
+            MaxValueValidator(
+                datetime.date(
+                    year=MONTH_NAVIGATION_FORM_MAX_YEAR_VALUE, month=MONTH_NAVIGATION_FORM_MAX_MONTH_VALUE, day=1
+                )
+            ),
+            MinValueValidator(
+                datetime.date(
+                    year=MONTH_NAVIGATION_FORM_MIN_YEAR_VALUE, month=MONTH_NAVIGATION_FORM_MIN_MONTH_VALUE, day=1
+                )
+            ),
+        ],
+    )
+
+    def __init__(  # pylint: disable=keyword-arg-before-vararg
+        self, initial_date: datetime.date = None, *args: Any, **kwargs: Any
```

Comment:
Was this solution meant to silence pylint? Because it still argues over it. ☹️ 
Mypy doesn't mind either way...

Reply:
well, it's weird - it looks like it doesn't complain on mine ;)
As to the root of the problem - `initial_date` is of `Optional[datetime.date]`, because it can be `None`: https://mypy.readthedocs.io/en/latest/kinds_of_types.html#optional-types-and-the-none-type